### ISSUE TITLE
Would it be possible to bypass the transfer dialog when pressing the xfr button?

### ISSUE THREAD

**Issue by videopilot** [DMagic1/KSP-Portrait-Stats#10]
It seems redundant to have to select a Kerbal to transfer when I've already pressed the xfr button on a specific portrait. Even more so when there is only one Kerbal on the vessel.

Would it be possible to just click the xfr button and then go straight to selecting a part?

**Comment by KerballOne** [comment]
Yes, was able to by recompiling the plugin after changing one line:	
		//CrewHatchController.fetch.SpawnCrewDialog(crew.InPart, false, true);
		CrewTransfer.Create(crew.InPart, protoCrew, null);

**Comment by Clayell** [comment]
This can probably be closed now
